Source/Core/BingMapsApi.js

```diff
@@ -35,14 +39,26 @@ define([
 
         if (!defined(BingMapsApi.defaultKey)) {
             if (!printedBingWarning) {
-                console.log('This application is using Cesium\'s default Bing Maps key.  Please create a new key for the application as soon as possible and prior to deployment by visiting https://www.bingmapsportal.com/, and provide your key to Cesium by setting the Cesium.BingMapsApi.defaultKey property before constructing the CesiumWidget or any other object that uses the Bing Maps API.');
+                console.log(errorString);
                 printedBingWarning = true;
             }
-            return 'Aj1ony_-Typ-KjG9SJWiKSHY23U1KmK7yAmZa9lDmuF2osXWkcZ22VPsqmCt0TCt';
+            return 'AnjT_wAj_juA_MsD8NhcEAVSjCYpV-e50lUypkWm1JPxVu0XyVqabsvD3r2DQpX-';
         }
 
         return BingMapsApi.defaultKey;
     };
 
+    BingMapsApi.getErrorCredit = function(providedKey) {
+        if (defined(providedKey) || defined(BingMapsApi.defaultKey)) {
+            return undefined;
+        }
+
+        if (!defined(errorCredit)) {
+            errorCredit = new Credit(errorString);
```

Comment:
That's not to say we can't tweak the wording if we want; it just all needs to be in the credit.